gbpservice/nfp/configurator/drivers/base/base_driver.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+from oslo_log import log as logging
+LOG = logging.getLogger(__name__)
+SUCCESS = 'SUCCESS'
+FAILED = 'FAILED'
+
+"""Every service vendor must inherit this class. If any service vendor wants
+   to add extra methods for their service, apart from below given, they should
+   add method definition here and implement the method in their driver
+"""
+
+
+class BaseDriver(object):
+    def __init__(self):
+        pass
+
+    def configure_interfaces(self, context, kwargs):
+        return SUCCESS
+
+    def clear_interfaces(self, context, kwargs):
```

Comment:
There are method names that are constructed and not exposed in blue print. What blue print talked about is resource name (interfaces/routes), not the method name.

We followed the convention of configure_* and clear_* inside the code as they make more sense.